app/views/post/index.html.erb

```diff
@@ -0,0 +1 @@
+<h1>List of all post for a specific user</h1>
```

Comment:
- [ ] This change is for the folder containing this file, not the file itself. The folder should be `posts` to match the controller, not `post`.